test/map/transform.test.cpp

```diff
@@ -109,7 +109,13 @@ TEST(Transform, IntegerZoom) {
     auto checkIntegerZoom = [&transform](uint8_t zoomInt, double zoom) {
         double scale = transform.getState().zoomScale(zoom);
         transform.setScale(scale);
+#if defined(__ANDROID__)
+        // Android uses log(x) / M_LN2 instead of log2(x) because the latter
+        // is broken in ARMv5 - that approach being less precise than log2(x).
+        ASSERT_FLOAT_EQ(transform.getScale(), scale);
+#else
         ASSERT_DOUBLE_EQ(transform.getScale(), scale);
+#endif
```

Comment:
You could use [`EXPECT_NEAR` instead](https://github.com/google/googletest/blob/master/googletest/docs/AdvancedGuide.md#floating-point-macros) and supply a custom epsilon value.